sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -328,7 +325,6 @@ def read_resource(res: 'CustomResource',
 
     async def do_read():
         try:
-            log.debug(f"preparing read: ty={ty}, name={name}, id={opts.id}")
```

Comment:
Doesn't really add much new information beyond what was logged at the beginning of the function, and Node.js doesn't have this additional logging.